bluemira/base/reactor_config.py

```diff
@@ -229,12 +226,51 @@ def _check_args_are_strings(self, args: Iterable[str]):
             if not isinstance(a, str):
                 raise ReactorConfigError("args must be strings")
 
+    def _expand_paths_in_dict(self, d: Dict[str, Any], rel_path: Path):
+        """
+        Expand all file paths in the dict
+        by replacing their values with the json file contents.
```

Comment:
The docstring summary should be on one line. This could be shortened a bit so it fits:

`Expand all file paths by replacing their values with the json file's contents.`